helper.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const Variables = require('sq-toolkit/variables');
+const Exception = require('sq-toolkit/exception');
+const BigQuery = require('@google-cloud/bigquery').BigQuery;
+
+let _instance = null;
+
+const ErrorCode = {
+    ERROR_INSTANCE_ALREADY_CREATED: 'ERROR_INSTANCE_ALREADY_CREATED',
+    ERROR_INSTANCE_NOT_CREATED: 'ERROR_INSTANCE_NOT_CREATED'
```

Comment:
I'd put this in other file to allow access from outside library (maybe for testing)

Reply:
This `ErrorCode` object is exposed at line `16`